todo_app/templates/index.html

```diff
@@ -18,9 +18,31 @@ <h1 class="display-4">To-Do App</h1>
 <br>
   <div class="row">
     <div class="col-md-12">
-       <h2>Items</h2>
+       <h2>To Do...</h2>
        <ul class="list-group mb-4">  
-        {% for item in items %}
+        {% for item in view_model.todo_items %}
+          <li>{{item.name}} - {{item.status}}</li>
+          {% if item.status != "Done"%}
+          <form action="complete-item/{{item.id}}" method="post">  
+             <button>Complete</button>
+             </form>
+             {% endif %}    
```

Comment:
We can drop these if checks now, since we know that everything in this list is in the to-do status by default!

Also, it doesn't change anything concrete, but it's worth trying to be strict about indentation, since it can make code significantly easier to read, e.g. if we weren't removing the if:
```python
          {% if item.status != "Done"%}
            <form action="complete-item/{{item.id}}" method="post">  
              <button>Complete</button>
            </form>
          {% endif %}    
```
or if we were:
```suggestion
          <form action="complete-item/{{item.id}}" method="post">  
             <button>Complete</button>
          </form>
```